plugins-bundled/internal/marketplace-app/src/components/Card.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { useTheme2, stylesFactory } from '@grafana/ui';
+
+interface Props {
+  href: string;
+  text: React.ReactNode;
+  image: React.ReactNode;
+  layout?: 'horizontal' | 'vertical';
+}
+
+export const Card = ({ href, text, image, layout = 'vertical' }: Props) => {
+  const theme = useTheme2();
+  const styles = getCardStyles(theme, layout);
+
+  return (
+    <a href={href} className={styles.root}>
+      <div className={styles.container}>
+        <div className={styles.imgContainer}>{image}</div>
+        {text}
+      </div>
+    </a>
+  );
+};
+
+const getCardStyles = stylesFactory((theme: GrafanaTheme2, layout) => ({
```

Comment:
Maybe remove all `stylesFactory` usage and use useStyles2 instead?

Reply:
I don't believe it's possible to pass arguments other than the theme to the `useStyles` hook?

Reply:
Ok, gotcha!

Reply:
I guess you can compose styles afterwards with the layout prop, but nevermind that now.

Reply:
That's true, I didn't think of that. Will deal with it in a future PR. Thanks @hugohaggmark!